kani-compiler/src/kani_middle/reachability.rs

```diff
@@ -83,6 +83,14 @@ where
     crate_items
         .iter()
         .filter_map(|item| {
+            // avoid stable MIR panic
+            // https://github.com/rust-lang/project-stable-mir/issues/95
```

Comment:
Can you file a tracking issue in Kani to remove this once the upstream issue is fixed and refer to it here?

Reply:
done